src/util/offscreen_canvas_mangled.ts

```diff
@@ -0,0 +1,31 @@
+import {offscreenCanvasSupported} from './offscreen_canvas_supported';
+
+let manglesOffscreenCanvas: boolean;
+
+export function offscreenCanvasMangled(): boolean {
+    if (manglesOffscreenCanvas == null) {
+        manglesOffscreenCanvas = false;
+        // browser can mangle canvas#getImageData results when enhanced privacy protectsion are enabled (see #85)
```

Comment:
```suggestion
        // browser can mangle canvas#getImageData results when enhanced privacy protection is enabled (see #3185)
```

Or did you mean `#3110`?